src/select/styled-components.js

```diff
@@ -60,40 +60,16 @@ export const InputContainer = styled('div', props => {
   };
 });
 
-export const Tag = styled('span', props => {
-  const {$theme, $multiple} = props;
-  const {
-    colors: {primary400, mono400},
-    sizing: {scale800, scale0},
-    typography: {font200},
-  } = $theme;
-  return $multiple
-    ? {
-        ...font200,
-        height: scale800,
-        fontWeight: 'bold',
-        display: 'flex',
-        alignItems: 'center',
-        paddingTop: scale0,
-        paddingBottom: scale0,
-        paddingRight: '5px',
-        paddingLeft: '5px',
-        margin: '5px',
-        borderWidth: '1px',
-        borderColor: primary400,
-        color: primary400,
-        borderRadius: '7px',
-        backgroundColor: mono400,
-      }
-    : {
-        ...getInputStyles({...props, $size: SIZE.default, $disabled: true}),
-        cursor: 'pointer',
-        width: 'auto',
-        flexGrow: '1',
-        ':hover': {
-          cursor: 'pointer',
-        },
-      };
+export const SingleSelection = styled('span', props => {
+  return {
+    ...getInputStyles({...props, $size: SIZE.default, $disabled: true}),
+    cursor: 'pointer',
+    width: 'auto',
+    flexGrow: '1',
+    ':hover': {
+      cursor: 'pointer',
```

Comment:
I'm curious why we need to specify `cursor` again in `:hover`.

Reply:
pne of the ways to override the same style from original Input. This is the bug about

Reply:
Input has been updated on master, can we fix this?

Reply:
ok